README.md

```diff
@@ -59,7 +92,7 @@ And accessible at the following URL:
     * `digest_ie`
     * `ntlm_wb`
 
-##### SCC settings for accessing SUSE repositories
+### SCC settings for accessing SUSE repositories
 
 The `scc` section contains your organization credentials for mirroring SUSE repositories.
 Your organization credentials can be obtained at [SUSE Customer Center](https://scc.suse.com).
```

Comment:
obtained **from the** SUSE Customer Center...
Maybe point out where in the SCC the credentials are?